AngularInventory/src/styles.css

```diff
@@ -22,9 +22,24 @@ h1 {
 }
 
 .card {
-    max-width: 250px;
+    width: 500px;
+    box-shadow: 5px 10px 18px #888888;
 }
 
 .col {
-    max-width: 250px;
+    max-width: 800px;
+}
+
+.buttonimg {
+    width: 500px;
+}
+
+.edgeimg {
```

Comment:
edgeImg doesn't really mean anything, it would be hard to tell how to use this based on the name. Consider renaming it or leaving a comment to explain what it is.